client/src/data/locales/en-US.json

```diff
@@ -1,14 +1,24 @@
-  "home.quote": "...working to create a more equitable restaurant ecosystem for all..."
+  "header.maps": "MAPS",
+  "header.database": "LICENSE DATABASE",
+  "header.resources": "LICENSE RESOURCES",
+  "header.home": "HOME",
+  "navigation.maps": "MAPS",
+  "navigation.database": "LICENSE DATABASE",
+  "navigation.resources": "LICENSE RESOURCES",
+  "hero.heading": "Navigating the Liquor License system in Boston is hard.{br}We are here to make it just a little bit easier.",
+  "hero.photoCredit": "Photo by {photographerLink} on {platformLink}",
+  "aboutUs.title": "Who We Are:",
+  "aboutUs.weAre": "We are a collection of restaurant owners, bartenders, chefs, and patrons; <b>Just Like You.</b> We understand how hard it is to navigate the red tape of liquor licenses in Boston. From tracking availability to the potential unwritten costs of obtaining one. We've made it our mission to demystify the process to help you get your business off the ground.",
+  "home.quote": "...working to create a more equitable restaurant ecosystem for all...",
+  "license-type.h2": "What does each type of license mean?",
+  "license-type.h2-p": "Boston has several different types of licenses from a variety of different laws and statues, which can lead to some confusion when applying for a license. These licenses can range from those that can be applied city-wide to licenses for specific areas of Boston. Below are some basic definitions of each license type to aid you on your search",
+  "license-type.transferable-h3": "Transferable Licenses",
+  "license-type.transferable-p": "Licenses that have no restrictions preventing them from being transferred to another business and/or another part of the city of Boston. Most of the Licenses in Boston are made of this type and all Licenses issued before 2006 will fall under this category.",
+  "license-type.nonTransferable-h3": "Non-Transferable Licenses",
+  "license-type.nonTransferable-p": "Restrictions are placed on certain licenses preventing them from being transferred outside of their legally designated area of Boston. These licenses can come in several different forms, most notably the licenses created under the 2024 law sedignating Licenses to Several Zip Codes.",
+  "license-type.law-h4": "New 2024 Law",
+  "license-type.law-p": "In 2024 the city of Boston passed a new law to increase access to lquor licenses in several parts of Boston. Through this law the city will be issuing 5 new on-premise liquor licenses to 13 of Boston's Zip Codes each year for the next 3 years.",
+  "license-type.specialAreas-h4": "Special Areas of Boston",
+  "license-type.specialAreas-p": "Another provision of the 2024 law was the creation of specialty licenses for Oak Square in Brighton, several other legislations in the past have also carved out licenses for specific areas in the city as well."
```

Comment:
🌴 Non-blocking comment:
You could make these easier to find for translators if you nest them like this: `database-page.license-type.header2`

better yet would be to nest them this way:
```
"database-page": {
  "license-type": {
    "header2": "What does each type of license mean?"
  }
}
```

Reply:
haha, thanks for trying 😅